src/platforms/dart/usage/advanced-usage.mdx

```diff
@@ -7,6 +7,51 @@ sidebar_order: 2
 
 - For the usage of the Dart SDK, the minimal required Dart SDK version is `2.8.0` and Flutter SDK version is `1.17.0`
 
+## Automatic Breadcrumbs
+
+If you want to track automatic [Breadcrumbs for HTTP requests](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types), you can add a Sentry wrapper that does it automatically for you.
```

Comment:
```suggestion
To track automatic [breadcrumbs for HTTP requests](https://develop.sentry.dev/sdk/event-payloads/breadcrumbs/#breadcrumb-types), you can add a Sentry wrapper that does this automatically.
```

Reply:
I'd rather keep `Breadcrumbs` and `it`, the rest makes sense to me